database/migrations/2023_05_03_000005_create_plan_offers_table.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreatePlanOffersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up(): void
+    {
+        Schema::create(config('rinvex.subscriptions.tables.app_market_plan_offers'), function (Blueprint $table) {
+            $table->increments('id');
+            $table->integer('app_id')->unsigned();
```

Comment:
I think it is better to make it, unsigned big integer to matcha the `id` in apps table